### ISSUE TITLE
Mirror messages from Slack bots to IRC

### ISSUE THREAD

**Issue by jvperrin** [ocf/slackbridge#14]
The Jenkins bot, or some of the GitHub bot's messages should be mirrored to IRC (mostly in `#rebuild-spam`). This would be nice to give more context for users in that channel since Slack has more messages and some of them are useful, like builds failing.

**Comment by chriskuehl** [comment]
Why not just use Jenkins IRC integrations? All of these things also integrate with IRC and the UX will probably be better than a bridged message (e.g. it can use colors).

**Comment by gnowxilef** [comment]
+1 to what @chriskuehl said

**Comment by jvperrin** [comment]
Yeah, that works too, there's bound to be some integrations that don't have IRC counterparts though, like the newly added `decalbot` that posts whenever someone signs up for the decal in `#decal`.

I agree for Jenkins it would be fine to just use the IRC integration instead though, and colors would be nice to have (providing #1 is fixed).

**Comment by jvperrin** [comment]
@gnowxilef There's a emoji react for that in the top right of the message now instead of commenting (:+1:)

**Comment by gnowxilef** [comment]
too many new things.

**Comment by jvperrin** [comment]
lol agreed, and thanks @chriskuehl for the downvote :P

**Comment by gnowxilef** [comment]
countering the downvote.

**Comment by dkess** [comment]
Does anyone know if this is still an issue?

**Comment by jvperrin** [comment]
It is, github integration messages in #rebuild-spam for instance aren't mirrored because they don't have a corresponding IRC bot.
